### DISCUSSION TITLE
log in

### DISCUSSION THREAD

**Discussion by JheremeiArciaga** [vercel/community#3557] [Account Issue] [answered]
### Question

what's the problem of that log in ? its always said that i need to go back 12 hrs even 2 days after they say it

**Comment by amyegan** [comment, marked as answer]
Please email [registration@vercel.com](mailto:registration@vercel.com) and include the email and username for the account. Vercel employs multiple layers of security and verification. Providing additional information about your account and the project you are deploying can help accelerate this process.
